src/cljc/rems/common/roles.cljc

```diff
@@ -32,7 +33,29 @@
             (boolean (some (set roles) context/*roles*)))
      :cljs (some (set roles) (:roles @(rf/subscribe [:identity])))))
 
+(defn is-with-organization?
+  "Determines if the user is part of an organization based on the passed in ID."
+  [organization-id]
+  #?(:cljs (let [owned-organizations @(rf/subscribe [:owned-organizations])]
```

Comment:
Subscribing here, in a helper function could be slightly risky. It creates a dependency for anyone using this utility to the re-frame subscription, which is not so apparent, and may cause the "wrong component" to be re-rendered when the value changes. We could pass here the owned organizations as a parameter to make this a pure utility function. 

I think the whole `in?` we can reduce to `(contains? (set (map :organization/id owned-organizations) organization-id)` which I don't think we need a utility for. If we so like we could create a subscription for it instead.

Reply:
Oh one more thing as a note we could record for the future as a task. We should probably make sure that the logic is correct wrt. enabled and disabled organizations. Perhaps organization ownership is OK to exist even for a disabled organization, and you could plausibly edit things in the disabled organization. But a disabled organization should not show up as a valid option for "end users". We could have some old bugs related to this.